apps/api/v2/src/modules/organizations/organizations.module.ts

```diff
@@ -130,6 +134,9 @@ import { Module } from "@nestjs/common";
     ZoomVideoService,
     Office365VideoService,
     TokensRepository,
+    TeamsVerifiedResourcesRepository,
+    WorkflowsRepository,
+    TeamWorkflowsService,
```

Comment:
The `organizations.module.ts` is huge - can we have:
1. workflows module with the repository
2. team workflows module that then imports workflows repository if needed
3. organization team workflows module that right now would expose the new controller

And then `organizations.module.ts` imports the organizaiton team workflows module or we simply directly just import it in the endpoints module or wherever the organizations module is imported in.

Reply:
PS At some point would be nice to break down the organizations module into smaller modules because it has so many imports that we often cant import organizaitons module where we need because it leads to circular dependencies.